src/Uno.UWPSyncGenerator/DocGenerator.cs

```diff
@@ -140,6 +140,18 @@ void BuildMemberLists()
 									_sb.AppendParagraph($"In addition, {formattedViewName} has Uno-specific documentation {Hyperlink("here", customDocLink)}.");
 								}
 
+								var galleryLink = GetGalleryLink(viewName);
```

Comment:
@rafael-rosa-knowcode  For some reason, it does not look that the link is generated properly: https://unodocsprstaging.z13.web.core.windows.net/pr-14774/articles/implemented/microsoft-ui-xaml-controls-combobox.html

ComboBox is in the list of gallery/playground links but it does not appear in the generated doc.

Reply:
My mistake. It was in the wrong branch. The other is still in progress.
Reverted..